cookbooks/bcpc/attributes/apache2.rb

```diff
@@ -13,6 +13,10 @@
 
   # extra files to be exposed
   bach_web['files_url'] = 'files'
+
+  # links to services (an hash of hashes with structure:
+  # <name>=>{'desc':<desc> and 'url':<url>} to be listed)
+  bach_web['links'] = []
```

Comment:
Thanks for adding the default value. 
I think the comment is correct, and it should be `{}` rather than [] [(reference here)](https://github.com/bloomberg/chef-bach/blob/15b957b207aa05f9ea8a066d5351162e1c3f57b8/cookbooks/bcpc/templates/default/bach.html.erb#L21)

Although `service_ports`, and `files` are set by hdfs and hbase attribute file, I might be safer to set the default value here as well, adding
```
bach_web['service_ports'] = {}
bach_web['files'] = {}
```